File: app/src/main/java/com/stormbreakerstudios/android/challenge/MainActivity.java
package com.stormbreakerstudios.android.challenge;

import android.os.Bundle;

import androidx.appcompat.app.ActionBar;

import com.stormbreakerstudios.android.challenge.databinding.ActivityMainBinding;

public class MainActivity extends BaseActivity {

    ActivityMainBinding viewBindingMainBinding = null;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        viewBindingMainBinding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(viewBindingMainBinding.getRoot());
        setSupportActionBar(viewBindingMainBinding.toolbar);
        ActionBar bar = getSupportActionBar();
        if (bar != null)
            getSupportActionBar().setTitle(R.string.peace_time);
        viewBindingMainBinding.rvMain.setAdapter(new MainAdapter());
    }
}

File: app/src/main/java/com/stormbreakerstudios/android/challenge/MainActivity.kt
package com.stormbreakerstudios.android.challenge

import android.os.Bundle
import com.stormbreakerstudios.android.challenge.databinding.ActivityMainBinding

class MainActivity : BaseActivity() {
    lateinit var viewBindingMainBinding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        viewBindingMainBinding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(viewBindingMainBinding.root)
        setSupportActionBar(viewBindingMainBinding.toolbar)
        supportActionBar?.setTitle(R.string.peace_time)
        viewBindingMainBinding.rvMain.adapter = MainAdapter()
    }
}